dev/com.ibm.ws.security.acme/resources/com/ibm/ws/security/acme/resources/AcmeMessages.nlsprops

```diff
@@ -262,7 +263,7 @@ CWPKI2056W=CWPKI2056W: The validity period of the certificate with {0} serial nu
 CWPKI2056W.explanation=The validity period is shorter than the minimum renew time. The certificate expires before a new certificate is requested.
 CWPKI2056W.useraction=To avoid certificate expiration, request a certificate with a longer validity period. If the certificate authority supports a custom validity period, set the validFor property in the server configuration.
 
-CWPKI2057E=CWPKI2057E: Certificate revocation status checking didn''t create a CertPathValidator to validate the certificate. The error is ''{0}''.
+CWPKI2057E=CWPKI2057E: Certificate revocation status checking did not create a CertPathValidator to validate the certificate. The error is ''{0}''.
```

Comment:
`Certificate revocation status checking did not create a CertPathValidator to validate the certificate.`

what is CertPathValidator ? Is it a property?, If so

`Certificate revocation status checking did not create a value for the CertPathValidator property to validate the certificate.`

Reply:
Updated it -- it refers to constructing a java.security.cert.CertPathValidator instance